### ISSUE TITLE
problem "Expected the service to be RUNNING, but the service has FAILED"

### ISSUE THREAD

**Issue by wjcarpenter** [erlang/erlide_eclipse#295]
I have been wrestling with the same issue described in closed issue #214. I've tried the suggestions in that thread (and several more things I thought of myself) but did not get cured. To keep things isolated, I am running erlide in a fresh install of eclipse neon with Java 8 on ubuntu 14.04.

My simple erlang project (just one source file) shows 5 instances of this error as soon as I start eclipse.

**_inconsistent project Expected the service to be RUNNING, but the service has FAILED: java.lang.IllegalStateException_**

When I try to launch the application from that erlang project, it fails with this message:

**_An internal error occurred during: "Launching hello_world".
Expected the service to be RUNNING, but the service has FAILED_**

I have no ~/.erlide.hosts file, and I have no erlide-specific entries in my eclipse.ini. My host's name is "u1404", I'm running under an ordinary userid "wjc", and the first couple of lines of my /etc/hosts are:

**_127.0.0.1	localhost
127.0.1.1	u1404_**

My eclipse and erlide log files are attached.
[eclipse.log.txt](https://github.com/erlang/erlide_eclipse/files/721498/eclipse.log.txt)
[erlide.log.txt](https://github.com/erlang/erlide_eclipse/files/721499/erlide.log.txt)

**Comment by wjcarpenter** [comment]
Additional info... The preferences erlang -> Troubleshooting panel shows all empty values in the "-name" column. All of the "-sname" values are "u1404" except for ~/.erlide.hosts, which is empty. Both boxes in the ~/.erlide.hosts row are shaded red.

**Comment by vladdu** [comment]
Do you get the IDE working? Syntax highlighting, completion, etc? If I understand correctly, only trying to execute the project fails. For some reason, the node name in that case is empty. Try this: go to Run->Run configurations... and find an "erlang application" entry (should probably be only one). Make sure there is a node name on the "runtime" tab. Try to launch that again.

I that doesn't help, or if even the IDE doesn't work, we'll try debugging it later.

**Comment by wjcarpenter** [comment]
The IDE parts seem to be working OK.

The node name was empty, as was the cookie name. I put something in for both, but it didn't make any difference. Should those things be any specific values? (I used "u1404" for the node and "cookie" for the cookie.)

**Comment by wjcarpenter** [comment]
I think it must be something Neon-specific. I installed Eclipse Mars the same way, did the same steps, and it works as expected.

I'm happy to gather any data if you want look into this further, but no need to pursue it for me. I'll just use Mars.

**Comment by vladdu** [comment]
Strange, there should be nothing specific to the Eclipse version. I fixed the empty node name issue, thanks for reporting that.

BTW, did you start your application (the first time) by right-clicking on project and choosing Run As -> Erlang application? Or did you create a run configuration from scratch? If you did the latter, and if you still have the Neon instance, could you please try the formar alternative, to see if it works better? (Creating the launch manually may miss some important data)

**Comment by wjcarpenter** [comment]
I didn't remember my steps to create it, so I deleted the erlang project (the hello_world tutorial example) and recreated it. When I create the run configuration with "Run As -> Erlang application", it's all good. When I create it from scratch from the Run Configurations panel, it shows the symptoms I described. 

So, I think you theory is correct.

Then I went to Mars and got the same results, so you were right that it's not Neon-specific.
